src/components/layout/header/Header.jsx

```diff
@@ -72,7 +72,10 @@ const Header = () => {
                 toggleMenu && 'govuk-header__navigation-list--open'
               }`}
             >
-              <li className="govuk-header__navigation-item govuk-header__navigation-item govuk-!-margin-right-5">
+              <li
+                data-testid="header-home-link"
```

Comment:
Make sure you have the latest main merged in, all data-testid should be id fields currently